src/main/scala/com/workday/warp/persistence/mysql/HasWarpSlickDsl.scala

```diff
@@ -52,6 +52,72 @@ trait HasWarpSlickDsl {
       expression.apply(timestamp)
     }
 
+    /**
+      *  Translates to YEAR() function
+      *  @return just the year in Int
+      */
+    def year(): Rep[Int] = {
+      val expression = SimpleExpression.unary[Timestamp, Int] { (timestamp, queryBuilder) =>
+        queryBuilder.sqlBuilder += " YEAR ("
+        queryBuilder.expr(timestamp)
+        queryBuilder.sqlBuilder += ")"
+      }
+      expression.apply(timestamp)
+    }
+
+    /**
+      * Translates to DATE() function
+      * @return just date in "yyyy-MM-dd"
+      */
+    def date(): Rep[String] = {
+      val expression = SimpleExpression.unary[Timestamp, String] { (timestamp, queryBuilder) =>
+        queryBuilder.sqlBuilder += " DATE ("
+        queryBuilder.expr(timestamp)
+        queryBuilder.sqlBuilder += ")"
+      }
+      expression.apply(timestamp)
+    }
+
+    /**
+      * UNIX_TIMESTAMP operation - seconds elapsed since 1970-01-01 00:00:00 UTC
```

Comment:
It would be good to note for MySQL functions with optional/multiple arguments which mapping(s) are implemented. For example, the `UNIX_TIMESTAMP` mysql function can take in an optional `Date`, which may be a `Date`, `DateTime`, or `Timestamp`. See: https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_unix-timestamp

I would leave some more documentation on which mapping(s) are implemented. For mappings that aren't covered, I would leave a TODO for them